deployment/logs-benchmark/readme.md

```diff
@@ -1,23 +1,44 @@
 # Benchmark for VictoriaLogs
 
+Benchmark compares VictoriaLogs with ELK stack and Grafana Loki.
```

Comment:
Can we be consistent with the docs, have general readme with loki and elk or have a readme per benchmark
I recommend to keep everything in one readme as far as we have only elk and loki